tests/test_cli.py

```diff
@@ -37,3 +40,47 @@ def test_info_command_prints_metadata(example_geff_path):
 def test_main_invalid_command():
     result = runner.invoke(app, ["invalid"])
     assert result.exit_code != 0
+
+
+@pytest.mark.parametrize(
+    "is_gt,tczyx",
+    list(itertools.product([True, False], [True, False])),
+)
```

Comment:
```suggestion
@pytest.mark.parametrize("is_gt", [True, False])
@pytest.mark.parametrize("tczyx", [True, False])
```

same as above (pytest naturally gives you a cartesian product if you use two params)